docs/component-stability.md

```diff
@@ -257,6 +257,12 @@ To move within the 'Maintained' ladder ("graduate"), the process for doing so is
 3. If approved, a PR to change the stability level should be opened and MUST be approved by all
    listed code owners.
 
+## Deprecation Information
+
+When a component is moved to deprecated, a deprecation section should indicate the date it was deprecated
+as well as any migration guidance. In some occasions it might not offered migration guidance but reviewers should
```

Comment:
```suggestion
as well as any migration guidance. In some occasions it might not offer migration guidance but reviewers should
```

Reply:
Or `In some occasions it might not be offered migration guidance` (`be offered`)?